internal/impl/snowflake/integration_test.go

```diff
@@ -36,6 +43,51 @@ func EnvOrDefault(name, fallback string) string {
 	return value
 }
 
+var config struct {
+	db             string
+	schema         string
+	account        string
+	role           string
+	user           string
+	privateKeyFile string
+	privateKey     string
+	dsn            string
+}
+
+func ReplaceConfig(s string) string {
+	return strings.NewReplacer(
+		"$USER", config.user,
+		"$ACCOUNT", config.account,
+		"$DB", config.db,
+		"$ROLE", config.role,
+		"$SCHEMA", config.schema,
+		"$PRIVATE_KEY_FILE", config.privateKeyFile,
+		"$PRIVATE_KEY", config.privateKey,
+		"$DSN", config.dsn,
+	).Replace(s)
+}
+
+var setup = sync.OnceFunc(func() {
```

Comment:
I wonder if having a `func TestMain(m *testing.M)` here would help. Then again, I'm not really sure why this func must run only once (would be handy to have a comment).

Reply:
Yeah actually it's probably fine to have it just run everytime you call setup. I'll make that change 👍 Thanks @mihaitodor !